syz-manager/manager.go

```diff
@@ -981,6 +1030,14 @@ func (mgr *Manager) hubSync() {
 		delete(mgr.hubCorpus, sig)
 		a.Del = append(a.Del, sig.String())
 	}
+	for _, repro := range mgr.repros {
+		sig := hash.Hash(repro)
+		if mgr.hubRepros[sig] {
```

Comment:
I don't think we need mgr.hubRepros at all. We can just send diffs. I.e. send all mgr.repros and send mgr.repros to nil. Then next time will send just what was added to mgr.repros.